code-of-conduct.md

```diff
@@ -0,0 +1,3 @@
+# Kubernetes Community Code of Conduct
+
+Please refer to our [Kubernetes Community Code of Conduct](https://git.k8s.io/community/code-of-conduct.md)
```

Comment:
IMO we should directly link the CNCF Code of Conduct. The Kubernetes code of conduct is an indirect reference to the CNCF Code of Conduct:

```suggestion
Shipwright follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
```

Note that technically our slack channel has a separate Code of Conduct, though that ultimately links to the CNCF Code of Conduct.

Reply:
definitely, good point.